README.md

```diff
@@ -1,23 +1,30 @@
+<div align="center">
```

Comment:
Is this div required?

Reply:
No its not required... Its to center the some of the content on the top level readme.

Reply:
https://github.com/ONSdigital/eq-author-app/blob/404b392f25af745dd52ae90fa7e6dae5c4215fb5/README.md